metaci/build/flows.py

```diff
@@ -42,3 +46,7 @@ def post_task(self, step, result):
 
         if "robot_outputdir" in result.return_values:
             import_robot_test_results(flowtask, result.return_values["robot_outputdir"])
+            if settings.RESULT_EXPORT_ENABLED:
+                export_robot_test_results(
+                    flowtask.build_flow.build, result.return_values["robot_outputdir"]
+                )
```

Comment:
If this is going to export all test results associated with a build, then we should call it once after the build is done, not after each robot task. Otherwise builds which include multiple robot tasks will end up reporting the same test results more than once.